doc/user/mirrors.md

```diff
@@ -68,16 +57,16 @@ If `--full` is specified, the official image will be cloned intact.
 ### 3. Platform limitation
 
 If you only want to clone packages for a particular platform, you can use `-os` and `-arch` to qualify:
-- `tiup mirros <target-dir> ---os=linux`
-- Just want to clone amd64 architecture: `tiup mirros <target-dir> --arch=amd64`
-- Just want to clone linux/amd64: `tiup mirros <target-dir> --os=linux --arch=amd64`
+- `tiup mirror <target-dir> ---os=linux`
```

Comment:
I think it should be `tiup mirror clone <target-dir> ...`

Reply:
```suggestion
- `tiup mirror clone <target-dir> ---os=linux`
```